blogs/2022/05/18/dev-container-cli.md

```diff
@@ -0,0 +1,131 @@
+---
+Order: 73
+TOCTitle: Dev container CLI
+PageTitle: Development Container CLI
+MetaDescription: Provide a consistent development environment anywhere using the development container CLI.
+Date: 2022-05-18
+Author: Brigit Murtaugh
+---
+
+# The dev container CLI
+**A consistent environment anywhere with the dev container CLI**
+
+May 18, 2022 by Brigit Murtaugh, [@BrigitMurtaugh](https://twitter.com/BrigitMurtaugh)
+
+## TLDR;
+
+We’ve released a dev container CLI as part of the new Development Containers Specification. We’d love your feedback in [this issue](https://github.com/devcontainers/cli/issues/7), or new issues and PRs in the repo.
```

Comment:
Should we be clearer who "we" is? Is it the VS Code team, Microsoft, or a dev container consortium or community?

Reply:
Hmm that's a good point. We want to emphasize dev containers are an open spec for any user or tool to adopt, similar to LSP. In the [LSP announcement blog](https://code.visualstudio.com/blogs/2016/06/27/common-language-protocol), it starts with mentioning VS Code and then moves into using "we." 

Maybe we say "Our dev containers team at Microsoft and GitHub is excited to release a dev container CLI..."? How does that sound?

Reply:
Are there more than one team across the two companies?
"The dev container teams at Microsoft and GitHub are excited to release a dev container CLI..."

Reply:
Yeah I suppose we could consider them multiple teams. Updated to this wording.

Reply:
Hmm looking at it now, we say "dev containers" 3 times in this sentence 😄 Do you think "The dev container teams at Microsoft and GitHub are excited to release a **new** CLI as part of the new Development Containers Specification" makes sense?